README.md

```diff
@@ -267,7 +267,9 @@ e.g., details on the Many Independent Algorithm (MIO).
 
 ### License
 _EvoMaster_'s source code is released under the LGPL (v3) license.
-
+For a list of the used third-party libraries, you can directly see the root [pom.xml](./pom.xml) file.
+For a list of code directly imported (and then possibly modified/updated) from 
+other open-source projects, see [here](./docs/reused_code.md).
 
 
 ### ![](https://www.yourkit.com/images/yklogo.png)
```

Comment:
One of the libraries used by the regex distance code has BSD license, would that be compatible?